deployment/deployer.yaml

```diff
@@ -148,4 +172,51 @@ parameters:
   description: "Timeout for *expected* ops nodes to be present when cluster is recovering from a full restart."
   name: ES_OPS_RECOVER_AFTER_TIME
   value: "5m"
-
+-
+  description: "Node selector key for Elasticsearch cluster (key: value)."
+  name: ES_NS_KEY
```

Comment:
That was my original plan, but i didn't try something like `...=key=value'.  It may be that since I was trying to assign with 'key: value' I ended up writing it this way to match the yaml structure.

I'll try to assign the value that way... and also not confuse nomenclature :)